models/misogyny_logreg.joblib.dvc

```diff
@@ -0,0 +1,7 @@
+md5: 0f4a3aaa1c81c1b103aaca8ec726a88f
```

Comment:
you don't need the `.gitkeep` file if you already have some other files in a folder.

Reply:
also, really `md5`? :D